Microsoft.OneWeek.Hack.Microids.IoTDevice/Dockerfile

```diff
@@ -1,5 +1,16 @@
-FROM mcr.microsoft.com/dotnet/core/aspnet:3.0 AS runtime
+FROM mcr.microsoft.com/dotnet/core/sdk:3.0 AS build-env
 WORKDIR /app
-COPY published/** ./
 
-ENTRYPOINT ["dotnet", "/app/Microsoft.OneWeek.Hack.Microids.IoTDevice.dll"]
\ No newline at end of file
+COPY . ./
+
+RUN dotnet restore
+
+RUN dotnet publish -c Release -o out
```

Comment:
You might want to double-check that it doesn't try a nuget restore here. Consider adding --no-restore to suppress repeating that step.